libsolidity/parsing/Token.cpp

```diff
@@ -155,16 +155,15 @@ tuple<Token::Value, unsigned int, unsigned int> Token::fromIdentifierOrKeyword(s
 			) {
 				int n = parseSize(positionX + 1, _literal.end());
 				if (
-					0 < m && m < 256 &&
-					0 < n && n < 256 &&
+					m + n > 0 &&
```

Comment:
but without this change, we cannot make a fixed0x8 or a fixed8x0...and I've made a test in this latest iteration with your exact concern and it passes through it...